api/auth.py

```diff
@@ -327,10 +356,82 @@ def post(self):
                 'message': 'Error, user does not exist.',
             }
             return make_response(jsonify(responseObject)), 401
+
+
+class RoomReserved(MethodView):
```

Comment:
Can you please move this to `api/rooms.py`? It would be better if we keep all room related APIs there